### ISSUE TITLE
Redesign create user function

### ISSUE THREAD

**Issue by sjacks26** [albany-social-media-analysis/tweetView#20]
In [admin_authentication.py](https://github.com/albany-social-media-analysis/tweetView/blob/master/admin_authentication.py), replace [create_project_lead](https://github.com/albany-social-media-analysis/tweetView/blob/master/admin_authentication.py#L38) and [create_project_analyst](https://github.com/albany-social-media-analysis/tweetView/blob/master/admin_authentication.py#L81) with a single create_user function.

Project leads will be assigned in the backend by tool admin.
Project analysts are assigned using [assign_analyst.py](https://github.com/albany-social-media-analysis/tweetView/blob/master/assign_analyst.py).

**Comment by sjacks26** [comment]
I think it also makes sense to simplify the return statements to only return one object. Use `\n` and `\t` if needed for formatting text.
If a more complex returned object makes more sense, we might want to create an actual object to return (i.e., something that can have different attributes as appropriate).

**Comment by sjacks26** [comment]
Does this change mean that [this function](https://github.com/albany-social-media-analysis/tweetView/blob/master/admin_authentication.py#L128) will still be useful in the backend? Maybe it's how assign_analyst assigns a new role, or maybe it's how the tool admin will assign project lead roles.

**Comment by Javolfuller** [comment]
> Does this change mean that [this function](https://github.com/albany-social-media-analysis/tweetView/blob/master/admin_authentication.py#L128) will still be useful in the backend? Maybe it's how assign_analyst assigns a new role, or maybe it's how the tool admin will assign project lead roles.

No, this will very well be useful. I'm thinking similarly to how we envisioned that a new user can search for projects to join, project leads on a project can search for users to assign to their project.
